internal/router/rpcservice/rpcservice_test.go

```diff
@@ -0,0 +1,161 @@
+package rpcservice
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/go-tableland/internal/tableland"
+	"github.com/textileio/go-tableland/mocks"
+)
+
+func TestRunReadQueryManyRows(t *testing.T) {
```

Comment:
Wanted to test `RunReadQuery` via JSON RPC calls just to be sure the wires were connecting properly in the RPC service and that the formatting and surrounding logic is working as expected.

Reply:
Nice! The mock library is very handy